csharp-basics/exercises/Loops/Loops/Exercise6/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Exercise6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Max value? ");
+            var max = Convert.ToInt32(Console.ReadLine());
+            for (int i = 1; i <= max; i++)
+            {
+                if (i % 3 == 0 | i % 5 == 0)
+                {
+                    if (i % 3 == 0)
```

Comment:
Kāpēc šis ir jāliek iekšā if un pēc tam vēlreiz if? Pietiek ar vienu

Reply:
Jo var būt gadījumi, kad jāprintē vai nu viens, vai nu otrs, vai pat abi
Tādēļ ir nested if's

Reply:
Vai arī tas kaut kādā ziņā nav pareizi?

Reply:
Un vai tad tas pats nenotiek bez i % 3 == 0 | i % 5 == 0? Un vispār ||, nevis |

Reply:
Un nespied request a review. Es jau nospiedu requested changes, bet tagad to rāda tikai history. + par katru komentāru, push  vai ko citu, man un Edgaram nāk e-pasti

Reply:
oki, nākamreiz zināšu

Reply:
> Un vai tad tas pats nenotiek bez i % 3 == 0 | i % 5 == 0? Un vispār ||, nevis |

pirmais if uzzin, vai ir jāraksta vārds/vārdi skaitļa vietā (kas ir pie else nosacījuma) nākamie divi if ir katram gadījumam, jo var gadīties, ka ir jāprintē abi, ne tikai viens vai otrs

Reply:
un nomainīju uz ||

Reply:
Ja ir jāvari izprintēt abus, to var izdarī ar if pie abiem